pkg/file/test-fixtures/generators/fixture-1.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 set -ue
 
+realpath() {
+    [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
```

Comment:
using `$PWD` would break this depending where this is running from, right? Are we trying to get the absolute path based of the directory where this script exists? or something else?

Reply:
I defer to @wagoodman ^

Reply:
In short, yes. But that is OK.

This is roughly the same behavior as the `realpath` tool:

```
is this an absolute path? if so just return it... otherwise, take the given relative path and resolve it to an absolute path relative to the current working directory
```

So yes, depending on where you run this, the result will be wrong. Given that go will be running this script, it is always relative to the test directory in question it should always give the correct path.

(side note, somewhat unrelated... swapping `$PWD` for `$(pwd)` would be a touch better)